src/react/reconcilation.js

```diff
@@ -891,6 +891,26 @@ export class Reconciler {
     branchState: BranchState | null,
     evaluatedNode: ReactEvaluatedNode
   ): AbstractValue {
+    let closureVal;
+    let thisVal;
+
+    invariant(this.realm.generator);
+    if (value.kind === "(A).map(B,C)") {
+      let args = this.realm.generator.getArgsForDeclaredValue(value);
```

Comment:
I don't understand this. Why this roundabout way via `getArgsForDeclaredValue`, when I think the answer should be available as `value.args`? Except that `getArgsForDeclaredValue` will fail if the value comes from some nested generator.

Reply:
That would be ideal, but the args are empty as this a temporal abstract :(